frontend/package.json

```diff
@@ -9,6 +9,7 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "20": "^3.1.9",
```

Comment:
what is `"20": "^3.1.9",`?

Reply:
Most likely, when I installed git via xcode install.